.github/workflows/temp.yml

```
@@ -0,0 +1,59 @@
+      - synchronize # Trigger when commits are added to the pull request
+      - opened # Trigger when a pull request is first opened
+      - reopened # Trigger when a previously closed pull request is reopened
+
+jobs:
+  instrumentation-tests:
+    name: Run Instrumentation Tests
+    runs-on: ubuntu-latest
+    steps:
+      # Checkout the code
+      - name: Checkout
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)